backend/apps/content/management/commands/seed_lessons.py

```
@@ -292,50 +293,104 @@
+                                self.stdout.write(
+                                    self.style.SUCCESS(
+                                        f"  ✓ Created exercise: {ex_obj.title}"
+                                    )
+                                )
+                            else:
+                                self.stdout.write(
+                                    self.style.WARNING(
+                                        f"  ~ Skipped exercise: {ex_obj.title}"
+                                    )
+                                )
+            except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)